src/main/java/io/reactivex/parallel/ParallelFlowable.java

```diff
@@ -611,7 +611,7 @@ protected final boolean validate(Subscriber<?>[] subscribers) {
      * @return the value returned by the converter function
      */
     @CheckReturnValue
-    public final <U> U to(Function<? super ParallelFlowable<T>, U> converter) {
+    public final <U> U to(@NonNull Function<? super ParallelFlowable<T>, U> converter) {
         try {
             return converter.apply(this);
```

Comment:
I'd still check it in order to be consistent and for the better error message knowing what exactly is null.